main/src/ca/uwaterloo/flix/language/phase/Parser2.scala

```diff
@@ -1581,8 +1581,7 @@ object Parser2 {
              | TokenKind.TripleTilde
              | TokenKind.KeywordLazy
              | TokenKind.KeywordForce
-             | TokenKind.KeywordDiscard
-             | TokenKind.KeywordDeref => unaryExpr()
+             | TokenKind.KeywordDiscard => unaryExpr()
```

Comment:
Weird that `ref` and `deref` are not treated the same.

Reply:
Yeah. Not sure. But it does not matter now.